packages/auth/src/core/util/handler.ts

```diff
@@ -107,7 +114,18 @@ export function _getRedirectUrl(
       delete paramsDict[key];
     }
   }
-  return `${getHandlerBase(auth)}?${querystring(paramsDict).slice(1)}`;
+
+  // Sets the App Check token to pass to the widget
+  const appCheckToken = await auth._getAppCheckToken();
```

Comment:
What will happen if _getAppCheckToken() failed? Should we return a url without an app check token, or throw an exception?

Reply:
`_getAppCheckToken()` will generally return a token (either a real one or a dummy token, for more info see the comment in auth_impl.ts) and we will always pass that real / dummy token to the widget. If there's an error, we simply log it client side, and when the backend receives a dummy token, it would prevent the request from succeeding if App Check is enforced.